### PR TITLE
pfff needs older Menhir

### PR DESCRIPTION
**Pull request by Leonidas-from-XIV**
Bug reported on Discuss https://discuss.ocaml.org/t/10737

It might be reasonable to add the same constraint on all versions, since I assume the issue is valid for all releases.

### CODE REVIEW

**Comment by Leonidas-from-XIV** [comment]
Looks like it also needs some lower bound on Menhir but from the error message I can't tell what the lower bound should be.

File: packages/pfff/pfff.0.25.1/opam [hunk 1]
```diff
@@ -14,7 +15,9 @@ depends: [
   "camlp4"
   "num"
 ]
+bug-reports: "https://github.com/facebook/pfff/issues"
 dev-repo: "git+https://github.com/facebook/pfff"
+license: "LGPL-2.1-only WITH OCaml-LGPL-linking-exception"
```
Comment (kit-ty-kate) [inline packages/pfff/pfff.0.25.1/opam:20]: reading the `license.txt` file it seems to be the (default) `-or-later` variant
```suggestion
license: "LGPL-2.1-or-later WITH OCaml-LGPL-linking-exception"
```

Comment (kit-ty-kate) [reply packages/pfff/pfff.0.25.1/opam:20]: ah actually nevermind I'm dumb

**Comment by kit-ty-kate** [comment]
Thanks!

**Pull request merged**